### DISCUSSION TITLE
Cluster Specific Positive TF Regulators

### DISCUSSION THREAD

**Discussion by strawberry789** [GreenleafLab/ArchR#800] [Questions / Documentation]
Section 15.4 of the ArchR manual shows how to identify positive TF regulators and provides a list of such, but is it possible to identify cluster-specific positive TF regulators?

Thanks.

**Comment by rcorces** [comment]
Seems like you could get this from an intersection of TFs with cluster-specific deviations (identified with `getMarkerFeatures`) and TFs identified as positive regulators?

**Reply by strawberry789** [reply]
I checked that not every identified TF positive regulator appears in the list of cluster-specific deviations identified with `getMarkerFeatures`
